src/bioregistry/data/bioregistry.json

```diff
@@ -68080,7 +68080,7 @@
         "Phenotype"
       ],
       "name": "MedGen",
-      "pattern": "^(?CN?\\d{4,7})$",
+      "pattern": "^(?CN?\\d{3,7})$",
```

Comment:
This is within the `togoid` mapped block. I will move it to outside the block so it serves as a Bioregistry override.

Reply:
@matentzn small issue... the PR is from your `main` branch so I can't push to it. Would you mind reverting the change above inside the `togoid` block and instead adding a new key directly under the `medgen` key
```json
    "pattern": "^[CN]*\\d{3,7}$"
```